src/widgets/Stepper.js

```diff
@@ -53,8 +53,7 @@ const StepperNumber = ({ step, numberOfSteps, currentStep, title, onPress }) =>
   const lastStep = step === numberOfSteps - 1;
   const completedStep = currentStep > step;
   const Container = completedStep ? TouchableOpacity : View;
-
-  const wrappedOnPress = () => onPress(step - 1);
+  const wrappedOnPress = () => onPress(step);
```

Comment:
is this a fix? I'm unsure why the `step - 1` was a thing, when the `onPress(step)` looks correct to me

Reply:
uhh.. I can't remember..